xde/src/xde.rs

```diff
@@ -413,7 +413,11 @@ const ONE_SECOND: Interval = Interval::from_duration(Duration::new(1, 0));
 
 #[no_mangle]
 fn expire_periodic(port: &mut Arc<Port>) {
-    port.expire_flows(Moment::now());
+    // XXX The call fails if the port is paused; in which case we
```

Comment:
Is it an issue that this would also ignore the error when the port is _not_ paused? I'm not sure what we can do about it, other than log a message to the syslog or something, but it might be good to emit it somewhere at this point.

Reply:
This call can only fail when the port is paused, that's what my comment is trying to say. I think the better solution is to just remove the periodic when the port is in this state, but that's work that needs to happen in xde and isn't part of this PR.